Pythonwin/pywin/framework/startup.py

```diff
@@ -9,8 +9,25 @@
 # this runs OK.  Errors in imported modules are much better - the messages go somewhere (not any more :-)
 
 import sys
+import os
 import win32ui
 
+if not sys.argv:
+	# Initialize sys.argv from commandline. When sys.argv is empty list (
+	# different from [''] meaning "no cmd line arguments" ), then C
+	# bootstrapping or another method of invocation failed to initialize
+	# sys.argv and it will be done here. ( This was a workaround for a bug in
+	# win32ui but is retained for other situations. )
+	import win32api
+	from ctypes import wintypes, windll, c_int, POINTER
```

Comment:
This would the the first use of ctypes, and `CommandLineToArgv` seems like to would be useful in other ways - how do you feel about just exposing this to win32api?

Reply:
Indeed, had several use case over time.  shlex.split(cmd, posix=0) works correct only for simple cases.